practice-app/database.py

```diff
@@ -0,0 +1,9 @@
+'''
+
+@author Ismail Ata Inan
+
+'''
```

Comment:
Same here :)

Reply:
I removed it

Reply:
I think it is bugged. I made this comment before the removal :(